Python/compile.c

```diff
@@ -1534,19 +1539,20 @@ compiler_body(struct compiler *c, location loc, asdl_stmt_seq *stmts)
     // If there are annotations and the future import is not on, we
     // collect the annotations in a separate pass and generate an
     // __annotate__ function. See PEP 649.
-    if (!(c->c_future.ff_features & CO_FUTURE_ANNOTATIONS) &&
+    if (!(FUTURE_FEATURES(c) & CO_FUTURE_ANNOTATIONS) &&
          c->u->u_deferred_annotations != NULL) {
```

Comment:
But the code still uses the compiler_unit explicitly for deferred annotaitons.

Reply:
Will need to deal with that later. But I want to think through whether this should be exposed from the compiler, or we need to add a higher-level compiler function for what's going on here.